common/app/views/fragments/email/emailSignUp.scala.html

```diff
@@ -12,7 +12,7 @@
 @formId = @{ componentClass + "-email-sub-form" }
 @inputId = @{ componentClass + "-email-sub-input" }
 
-@wrapperClass = @{"email-sub js-email-sub" + " email-sub--" + componentClass }
+@wrapperClass = @{"email-sub js-email-sub" + " email-sub--" + componentClass + " js-email-sub--" + componentClass}
```

Comment:
Tip! You could use string interpolation, e.g. `s"js-email-sub--${componentClass}"`. Don't worry about updating it though!